manimlib/mobject/svg/svg_mobject.py

```diff
@@ -200,16 +207,16 @@ def apply_style_to_mobject(mob, shape):
         )
         return mob
 
-    def path_to_mobject(self, path):
+    def path_to_mobject(self, path: se.Path) -> VMobjectFromSVGPath:
         return VMobjectFromSVGPath(path, **self.path_string_config)
 
-    def line_to_mobject(self, line):
+    def line_to_mobject(self, line: se.Line) -> Line:
         return Line(
             start=_convert_point_to_3d(line.x1, line.y1),
             end=_convert_point_to_3d(line.x2, line.y2)
         )
 
-    def rect_to_mobject(self, rect):
+    def rect_to_mobject(self, rect: se.Rect) -> Rectangle | RoundedRectangle:
```

Comment:
`RoundedRectangle` is inherited from `Rectangle`, so no need for multiple types here.